frontend/pages/admin/IntegrationsPage/cards/MdmSettings/WindowsMdmPage/WindowsMdmPage.tsx

```diff
@@ -94,6 +97,7 @@ const WindowsMdmPage = ({ router }: IWindowsMdmPageProps) => {
   return (
     <MainContent className={baseClass}>
       <>
+        gg
```

Comment:
??

Reply:
🤦‍♂️ vim to top of page...thank you for catching!